java/common/common-task/src/main/java/sleeper/task/common/RunCompactionTasks.java

```diff
@@ -257,15 +257,15 @@ public static void main(String[] args) {
         int numberOfTasks = Integer.parseInt(args[1]);
         AmazonSQS sqsClient = AmazonSQSClientBuilder.defaultClient();
         AmazonS3 s3Client = AmazonS3ClientBuilder.defaultClient();
-        AmazonAutoScaling asClient = AmazonAutoScalingClientBuilder.defaultClient();
+        AutoScalingClient asClient = AutoScalingClient.create();
+        Ec2Client ec2Client = Ec2Client.create();
         try (EcsClient ecsClient = EcsClient.create()) {
```

Comment:
The new clients should be included in this try/finally so that they will be closed, like the EcsClient.